src/Cake.Issues/IssueBuilder.cs

```diff
@@ -26,50 +27,66 @@ public class IssueBuilder
         /// <summary>
         /// Initializes a new instance of the <see cref="IssueBuilder"/> class.
         /// </summary>
+        /// <param name="identifier">The identifier of the message.</param>
         /// <param name="message">The message of the issue in plain text format.</param>
         /// <param name="providerType">The type of the issue provider.</param>
         /// <param name="providerName">The human friendly name of the issue provider.</param>
         private IssueBuilder(
+            string identifier,
             string message,
             string providerType,
             string providerName)
         {
+#pragma warning disable SA1123 // Do not place regions within elements
+            #region DupFinder Exclusion
+#pragma warning restore SA1123 // Do not place regions within elements
+
+            identifier.NotNullOrWhiteSpace(nameof(identifier));
             message.NotNullOrWhiteSpace(nameof(message));
             providerType.NotNullOrWhiteSpace(nameof(providerType));
             providerName.NotNullOrWhiteSpace(nameof(providerName));
 
+            #endregion
+
+            this.identifier = identifier;
             this.messageText = message;
             this.providerType = providerType;
             this.providerName = providerName;
         }
 
         /// <summary>
-        /// Initiates the creation of a new <see cref="IIssue"/>.
+        /// Initiates the creation of a new <see cref="IIssue"/> with <paramref name="message"/>
+        /// as identifier.
         /// </summary>
+        /// <typeparam name="T">Type of the issue provider which has the issue created.</typeparam>
         /// <param name="message">The message of the issue in plain text format.</param>
```

Comment:
Add comment that the message is used as identifier.

Reply:
@eoehen This is already mentioned in the [summary](https://github.com/cake-contrib/Cake.Issues/pull/185/files/6bc7d431b7cffe83a1e2d49522766e15ab88cb89#diff-b8874c866d3ad7f81867331cf1a3c342R58-R59). Isn't this enough? This is the documentation of the parameter where the message is expected. Therefore IMHO having the behavior that the message is used as identifier documented in the summary makes sense.